### PR TITLE
Create hybrid2.md

### PR DESCRIPTION
**Pull request by ethernomad**
#### Project Abstract

This application is for a follow-up grant to the original Hybrid Block Explorer grant: https://github.com/w3f/Grants-Program/pull/1582

This grant application is specifically to improve the indexer component of the project, including increased robustness & configurability, reverse batch indexing, custom event parameters, database tuning and WSS API improvements.

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [x] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (bank details via email _or_ BTC, Ethereum (USDC/DAI) or Polkadot/Kusama (USDT) address in the application).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Comment by ethernomad** [comment]
I have read and hereby sign the Contributor License Agreement.

**Review by Noc2 — APPROVED** [review]
Thanks for the follow-up application. I'm generally happy to go ahead with it, but it would be nice if you could promote your solution better in the community and potentially also consider the treasury for a follow-up grant. At the moment very few people seem to be aware of the solution or use it.

**Comment by ethernomad** [comment]
> Thanks for the follow-up application. I'm generally happy to go ahead with it, but it would be nice if you could promote your solution better in the community and potentially also consider the treasury for a follow-up grant. At the moment very few people seem to be aware of the solution or use it.

Yes, I haven't been promoting it yet. That will change with this grant and then I'll switch to treasury funding.

**Review by semuelle — CHANGES_REQUESTED** [review]
Hi @ethernomad. Thank you for the application. The deliverables all seem reasonable to me. Just two requests:
1. As David already pointed out, ideally the project would be run by more than just you and the occasional grant. Please consider doing more outreach.
2. Please provide a registered address. If you prefer not to disclose it here, please [email us](mailto:grants@web3.foundation).

**Comment by ethernomad** [comment]
> Hi @ethernomad. Thank you for the application. The deliverables all seem reasonable to me. Just two requests:
> 
>     1. As David already pointed out, ideally the project would be run by more than just you and the occasional grant. Please consider doing more outreach.
> 
>     2. Please provide a registered address. If you prefer not to disclose it here, please [email us](mailto:grants@web3.foundation).

Thanks @semuelle

1. I will be heavily promoting the project. The indexer can be extremely useful for developers on all Substrate chains. I'm determined to make the hosted block explorer a major component in the Polkadot ecosystem, supporting all chains.

2. I just emailed my address.

File: applications/hybrid2.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+### Contact
+
+- **Contact Name:** Jonathan Brown
+- **Contact Email:** jbrown@acuity.social
+- **Website:** n/a
+
+### Legal Structure
+
+- **Registered Address:** n/a
```
Comment (takahser) [inline applications/hybrid2.md:77]:
```suggestion
- **Registered Address:** (shared privately)
```

**Review by takahser — APPROVED** [review]
@ethernomad thanks for sharing your permanent address, I confirm its receival. Happy to approve as well here, though I agree with @Noc2 and @semuelle that more outreach would be useful and for the future, considering Treasury funding would be good.

**Review by takahser — APPROVED** [review]
Re-approving.

**Review by Noc2 — APPROVED** [review]

**Comment by keeganquigley** [comment]
Hi @ethernomad how is M4 progress coming along?

**Comment by ethernomad** [comment]
> Hi @ethernomad how is M4 progress coming along?

Good - I'm working on a project that uses the indexer. This will provide publicity and ongoing funding after the grant is competed.

**Comment by ethernomad** [comment]
> Hi @ethernomad just checking in again on M4?

I've finished M4 now. I'll be submitting the milestone for review in the next couple of days.

**Pull request merged**